Public/Src/App/Bxl/AzureDevOpsListener.cs

```diff
@@ -0,0 +1,212 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics.ContractsLight;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using BuildXL.Utilities.Collections;
+using BuildXL.Utilities.Instrumentation.Common;
+using BuildXL.Utilities.Tracing;
+using BuildXL.ViewModel;
+
+namespace BuildXL
+{
+    /// <summary>
+    /// This event listener should only be hooked up with AzureDevOps optimzied UI is requested

```

Comment:
>with AzureDevOps optimzied UI is requested [](start = 53, length = 42)

nit: I feel like something is missing in this phrase... 
"with AzureDevOps `when(?)` optimzied UI is requested"